ifrs17/Import/Importers.ipynb

```diff
@@ -464,10 +464,22 @@
                 "\n    var dataNodeSingleParametersUndefined = dataNodesImported.Where(x => x != null &&",
                 "\n                                                                    !dataNodeParametersDefined.Contains(x) && ",
                 "\n                                                                    dataNodesDefined.SingleOrDefault(y => y.SystemName == x) is GroupOfInsuranceContract).ToHashSet();",
-                "\n    await DataSource.UpdateAsync( dataNodeStatesUndefined.Select(x => ",
-                "\n        new DataNodeState {DataNode = x, Year = args.Year, Month = DefaultDataNodeActivationMonth, State = State.Active, Partition = partition.Id}).ToArray() );",
-                "\n    await DataSource.UpdateAsync( dataNodeSingleParametersUndefined.Select(x => ",
-                "\n        new SingleDataNodeParameter {DataNode = x, Year = args.Year, Month = DefaultDataNodeActivationMonth, PremiumAllocation = DefaultPremiumExperienceAdjustmentFactor, Partition = partition.Id}).ToArray() );",
+                "\n    if ((dataNodeStatesUndefined?.Count ?? 0) > 0)",
```

Comment:
we prefer to use .Any() for this kind of checks:
dataNodeStatesUndefined?.Any() ?? false

should work

Reply:
Yes, I changed this.